app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,221 @@
+/* Global Styles */
+
+html,
+body {
+  height: 100%;
+}
+
+body {
+  padding-top: 42px; /* Required padding for .navbar-fixed-top. Remove if using .navbar-static-top. Change if height of navigation changes. */
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-weight: 400;
+}
+
+.navbar {
+  font-weight: 400;
+}
+
+.page-header {
+  border-bottom: none;
+  color: white;
+  float: right;
+  font-weight: 500;
+}
+
+.featured {
+  color: black;
+  float: none;
+}
+
+.main-image {
+  background: url("dew-drops-carnivorous-plant.jpg") no-repeat center center;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  -webkit-background-size: cover;
+  background-size: cover;
+  height: 362px;
+}
+
+#index-foods,
+#index-accessories,
+#index-plants {
+  background: no-repeat center center;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  -webkit-background-size: cover;
+  background-size: cover;
+  height: 200px;
+
+  .btn {
+    margin-top: 130px;
+  }
+}
+
+#index-foods {
+  background: url("frog-in-pitcher-plant.jpg") no-repeat center center;
+  background-size: 100%;
+}
+
+#index-plants {
+  background: url("venus_flytrap.jpg") no-repeat center center;
+  background-size: 100%;
+}
+
+#index-accessories {
+  background: url("planters.jpg") no-repeat center center;
+  background-size: 100%;
+}
+
+#index-foods,
+#index-accessories,
+#index-plants {
+  height: 200px;
+}
+
+.img-portfolio {
+  margin-bottom: 30px;
+}
+
+.img-hover:hover {
+  opacity: 0.8;
+}
+
+/* Home Page Carousel */
+
+.carousel {
+  height: 50%;
+}
+
+.carousel .item,
+.carousel .item.active,
+.carousel .carousel-inner {
+  height: 100%;
+}
+
+.carousel .fill {
+  width: 100%;
+  height: 100%;
+  background-position: center;
+  background-size: cover;
+}
+
+#ProductCarousel .carousel,
+#ProductCarouselSm .carousel {
+  margin-bottom: 0;
+  padding: 0 40px 30px 40px;
+}
+
+/* The controlsy */
+#ProductCarousel .carousel-control,
+#ProductCarouselSm .carousel-control {
+  height: 40px;
+  width: 40px;
+  background: none repeat scroll 0 0 #222222;
+  border: 4px solid #FFFFFF;
+  border-radius: 23px 23px 23px 23px;
+  margin-top: 90px;
+}
+
+#ProductCarousel .carousel-control.right,
+#ProductCarouselSm .carousel-control.right {
+  font-size: 20px;
+  right: -12px;
+  height: 40px;
+  width: 40px;
+  background: none repeat scroll 0 0 #222222;
+  border: 4px solid #FFFFFF;
+  border-radius: 23px 23px 23px 23px;
+  margin-top: 6%;
+  color: #fff;
+}
+
+#ProductCarousel .carousel-control.left,
+#ProductCarouselSm .carousel-control.left {
+  font-size: 20px;
+  left: -12px;
+  height: 40px;
+  width: 40px;
+  background: none repeat scroll 0 0 #222222;
+  border: 4px solid #FFFFFF;
+  border-radius: 23px 23px 23px 23px;
```

Comment:
Shorthand form for property `border-radius` should be written more concisely as `23px` instead of `23px 23px 23px 23px`